planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -253,12 +253,27 @@ void GoalPlannerModule::updateData()
     initializeOccupancyGridMap();
   }
 
-  updateOccupancyGrid();
+  resetPathCandidate();
+  resetPathReference();
+  path_reference_ = getPreviousModuleOutput().reference_path;
 
-  // set current road lanes, pull over lanes, and drivable lane
-  setLanes();
+  updateOccupancyGrid();
 
   generateGoalCandidates();
+
+  if (!isActivated()) {
```

Comment:
@kosuke55
could you add comment why early return when the module is activated?

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/5554/commits/36d4457d3318ed9207c8f94a6c8c21be6e5328ff#diff-13561094b202845230024ce2547446a6d04d30789dda6e9dbe8b08671149ee96R267